.travis.yml

```diff
@@ -130,6 +130,7 @@ matrix:
         - unzip ~/chromedriver_linux64.zip -d ~/
         - rm ~/chromedriver_linux64.zip
         - sudo install -m755 ~/chromedriver /usr/local/bin/
+        - yarn install --pure-lockfile
```

Comment:
should 'pip inst all' have resolved this already?

Reply:
why would pip resolve our js dependencies?

Reply:
i thought we did something in setup.py that compiled JS assets -- maybe im mistaken

Reply:
oh idk, @JoshuaRLi just refactored things, so he may have changed that, but in the meantime we need this

Reply:
@dcramer are you perhaps talking about [`BuildAssetsCommand`](https://github.com/getsentry/sentry/blob/master/setup.py#L113)? IIRC everything related to building JS stuff is in the `Makefile`, for example, [this](https://github.com/getsentry/sentry/blob/master/Makefile#L135) webpack call.

Reply:
Never mind, [I found the thing](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/distutils/commands/build_assets.py#L139) in setup.py that compiles JS stuff.